redbot/cogs/audio/audio.py

```diff
@@ -1909,6 +1930,14 @@ def _track_creator(player, position=None, other_track=None):
             track_obj[key] = value
         return track_obj
 
+    @staticmethod
+    def _thumbnail(track):
+        if "youtube" in track.uri:
+            for v, k in track._info.items():
```

Comment:
Why not
```py
if "youtube" in track.uri and "identifier" in track._info:
    return "https://img.youtube.com/vi/{}/mqdefault.jpg".format(track._info["identifier"])
```
or even
```py
    return "https://img.youtube.com/vi/{identifier}/mqdefault.jpg".format(**track._info)
```
The `return None` is redundant behavior.

These are acceptable... but I still prefer changing the library instead.

Reply:
After https://github.com/Cog-Creators/Red-Lavalink/pull/38 is merged and released, I'll make another PR to change audio to use it.